packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/message-thread.object-metadata.ts

```diff
@@ -0,0 +1,75 @@
+import { FieldMetadataType } from 'src/metadata/field-metadata/field-metadata.entity';
+import { RelationMetadataType } from 'src/metadata/relation-metadata/relation-metadata.entity';
+import {
+  ObjectMetadata,
+  IsSystem,
+  FieldMetadata,
+  IsNullable,
+  RelationMetadata,
+} from 'src/workspace/workspace-sync-metadata/decorators/metadata.decorator';
+import { BaseObjectMetadata } from 'src/workspace/workspace-sync-metadata/standard-objects/base.object-metadata';
+import { MessageChannelObjectMetadata } from 'src/workspace/workspace-sync-metadata/standard-objects/message-channel.object-metadata';
+import { MessageObjectMetadata } from 'src/workspace/workspace-sync-metadata/standard-objects/message.object-metadata';
+
+@ObjectMetadata({
+  namePlural: 'messageThreads',
+  labelSingular: 'Message Thread',
+  labelPlural: 'Message Threads',
+  description: 'Message Thread',
+  icon: 'IconMessage',
+})
+@IsSystem()
+export class MessageThreadObjectMetadata extends BaseObjectMetadata {
+  @FieldMetadata({
+    // will be an array
+    type: FieldMetadataType.TEXT,
+    label: 'External Ids',
+    description: 'Thread ids from the messaging provider',
+    icon: 'IconMessage',
+  })
+  @IsNullable()
+  externalIds: string;
```

Comment:
Should we name it externalId for now until we implement arrays? You will probably need to update your code later anyway since the type will change?